garak/generators/nim.py

```diff
@@ -67,6 +71,13 @@ def _call_model(
         assert (
             generations_this_call == 1
         ), "generations_per_call / n > 1 is not supported"
+
+        if self.vary_seed_each_call:
+            _config.run.seed = random.randint(0, 65535)
```

Comment:
```suggestion
        if self.vary_seed_each_call:
            self.seed = random.randint(0, 65535)
```

Prefer not to churn values in `_config` as a way of setting state for a generator.

Reply:
agree, amended